docs/core/deploying/index.md

```diff
@@ -39,86 +39,11 @@ There are also a few disadvantages:
 
 - It is possible for the .NET Core runtime and libraries to change without your knowledge in future releases. In rare cases, this may change the behavior of your app.
 
-### Deploying a framework-dependent deployment ###
+## Self-contained deployments (SCD)
 
-Deploying a framework-dependent deployment with no third-party dependencies simply involves building, testing, and publishing the app. A simple example written in C# illustrates the process. The example uses the [dotnet utility](../tools/dotnet.md) from the command line; however, you can also use a development environment, such as Visual Studio or Visual Studio Code, to compile, test, and publish the example.
+For a self-contained deployment, you deploy not only your app and any third-party dependencies, but the version of .NET Core that you build your app with. Creating an SCD does not, however, include the [native dependencies of .NET Core](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md) itself on various platforms (for example, OpenSSL on macOS), so these need to be installed before running the application.
```

Comment:
> dependencies, but the version

remove comma

... or another way to state this that kills the sentence-ending preposition is ...

> For a self-contained deployment, you deploy your app and any included third-party dependencies along with the version of .NET Core used to build the app.

Next sentence:
* In that position, "however" seems *more formal* than just starting with the word.
* "does not" :point_right: "doesn't"
* I would drop the word "itself"
* "need to be installed" :point_right: "must be present" (the dev doesn't need to do it if they're already there)
* application :point_right: app

> However, creating an SCD doesn't include ...